tox_pipenv/plugin.py

```diff
@@ -94,92 +76,46 @@ def tox_testenv_install_deps(venv, action):
         deps = venv.get_resolved_dependencies()
     basepath = venv.path.dirpath()
     basepath.ensure(dir=1)
-    pipfile_path = _clone_pipfile(venv)
-    args = [sys.executable, "-m", "pipenv", "install", "--dev"]
+    pipfile_path, pipfile_lock_path = _clone_pipfile(venv)
+
+    install_args = _env_install_args()
+    if install_args is None:
+        if pipfile_lock_path is not None:
+            # project provided a lock file, so sync deps
+            install_args = ["sync", "--dev"]
+        else:
+            install_args = ["install", "--dev"]
```

Comment:
Good point. The plug was passing --dev, so i kept it.

On further reflection, there's no good reason to imply --dev as a default, especially since a caller could easily opt-in to that now. (Future rewrite adds pipenv_install_opts as testenv ini keys too)